src/bundle/Resources/views/content/tab/translations/tab.html.twig

```diff
@@ -21,6 +21,9 @@
             <th></th>
             <th>{{ 'tab.translations.language_name'|trans|desc('Language name') }}</th>
             <th>{{ 'tab.translations.language_code'|trans|desc('Language code') }}</th>
+            {% if main_translation_switch %}
```

Comment:
You could just use the fact that form is passed to the template to display this column, you don't need additional flag for this.